openapi/components/schemas/Invoices/UpcomingInvoiceItem.yaml

```
@@ -0,0 +1,90 @@
+type: object
+readOnly: true
+properties:
+  id:
+    type: string
+    description: ID of the upcoming invoice item.
+    maxLength: 50
+    example: ii_0YVFDEQS2KCFTBN9HXWJFY55GV
+  isInterim:
+    description: Specifies if the line item gets into an interim invoice.
```

nitpick: 'gets into' may be a bit ambiguous 
```suggestion
    description: Specifies if the line item is added an interim invoice.
```